oss-internship-2020/libtiff/README.md

```diff
@@ -0,0 +1,20 @@
+# sapi-libtiff
+Copyright 2020 Google LLC.
+
+## Start use
+You should make sure the libtiff submodule is cloned.
+
+`git clone --recursive https://github.com/alexelex/sapi-libtiff`
+
+## Usage
+
+#### build:
+`mkdir -p build && cd build && cmake .. -DSAPI_ROOT=/path/to/sapi_root -DBUILD_SHARED_LIBS=OFF`
+
+`make -j 8`
```

Comment:
Markdown support actual code blocks:

```
     ```
        mkdir -p build && cd build && \
          cmake .. -DSAPI_ROOT=/path/to/sapi_root -DBUILD_SHARED_LIBS=OFF
        make -j8
    ```
```